app/src/main/java/org/ole/planet/myplanet/ui/team/teamMember/AdapterJoinedMember.kt

```diff
@@ -19,39 +20,61 @@ import org.ole.planet.myplanet.utilities.Utilities
 class AdapterJoinedMember(private val context: Context, private val list: List<RealmUserModel>, private val mRealm: Realm, private val teamId: String) : RecyclerView.Adapter<AdapterJoinedMember.ViewHolderUser>() {
     private lateinit var rowJoinedUserBinding: RowJoinedUserBinding
     private val currentUser: RealmUserModel = UserProfileDbHandler(context).userModel!!
+    private val profileDbHandler = UserProfileDbHandler(context)
     private val teamLeaderId: String? = mRealm.where(RealmMyTeam::class.java)
         .equalTo("teamId", teamId)
         .equalTo("isLeader", true)
         .findFirst()?.userId
 
-    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): ViewHolderUser {
-        rowJoinedUserBinding = RowJoinedUserBinding.inflate(LayoutInflater.from(context), parent, false)
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): ViewHolderUser { rowJoinedUserBinding = RowJoinedUserBinding.inflate(LayoutInflater.from(context), parent, false)
         return ViewHolderUser(rowJoinedUserBinding)
     }
 
     override fun onBindViewHolder(holder: ViewHolderUser, position: Int) {
-        rowJoinedUserBinding.tvTitle.text = if (list[position].toString() == " ") list[position].name else list[position].toString()
-        rowJoinedUserBinding.tvDescription.text = "${list[position].getRoleAsString()} (${RealmTeamLog.getVisitCount(mRealm, list[position].name, teamId)} ${context.getString(R.string.visits)})"
+        val user = list[position]
+        rowJoinedUserBinding.tvTitle.text = user.name ?: user.toString()
+        rowJoinedUserBinding.tvDescription.text = "${user.getRoleAsString()} (${RealmTeamLog.getVisitCount(mRealm, user.name, teamId)} ${context.getString(R.string.visits)})"
 
-        Glide.with(context)
-            .load(list[position].userImage)
```

Comment:
this is the line that show the image. why is it removed? is it replaced by something else?